### PR TITLE
modules/darwin/community-builder: add user (adisbladis)

### PR DESCRIPTION
**Pull request by adisbladis**

### CODE REVIEW

**Comment by zowoq** [comment]
Do you want sudo on this box? At the moment giving admins their own accounts with sudo is a bit awkward, easier for us to share the `hetzner` account for now.

https://github.com/nix-community/infra/blob/0d98e6f5841b7d2211735682da2f049599938962/modules/darwin/common/default.nix#L15-L19

**Comment by adisbladis** [comment]
> Do you want sudo on this box? At the moment giving admins their own accounts with sudo is a bit awkward, easier for us to share the hetzner account for now.

Added my key to the shared hetzner user.

File: modules/darwin/community-builder/users.nix [hunk 1]
```diff
@@ -118,6 +118,11 @@ let
       trusted = true;
       uid = 546;
     }
+    {
+      name = "adisbladis";
+      trusted = true;
+      uid = 547;
+    }
```
Comment (zowoq) [inline modules/darwin/community-builder/users.nix:125]:
```suggestion
```
Sorry if I wasn't clear, I meant for this to be removed and only use the hetzner user.

Comment (adisbladis) [reply modules/darwin/community-builder/users.nix:125]: Oh. That's also reasonable.

**Pull request merged**